openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -481,15 +497,17 @@ def build_template(
                                               process if version is created
 
         """
-        template_preset = self.get_template_preset()
-
-        if template_path is None:
-            template_path = template_preset["path"]
 
-        if keep_placeholders is None:
-            keep_placeholders = template_preset["keep_placeholder"]
-        if create_first_version is None:
-            create_first_version = template_preset["create_first_version"]
+        if any(value is None for value in [template_path,
+                                           keep_placeholders,
+                                           create_first_version]):
```

Comment:
Refering to this comment https://github.com/ynput/OpenPype/pull/5830/files#r1375924385
```suggestion
        if (
            template_pathis None
            or keep_placeholders is None
            or create_first_version is None
        ):
```